src/PowerShellEditorServices.Protocol/DebugAdapter/StackFrame.cs

```diff
@@ -61,19 +61,28 @@ public class StackFrame
             var sourcePresentationHint = 
                 stackFrame.IsExternalCode ? SourcePresentationHint.Deemphasize : SourcePresentationHint.Normal;
 
+            // When debugging an interactive session, the ScriptPath is <No File> which is not a valid source file.
```

Comment:
Hmmm, was this supposed to be part of this PR?

Reply:
Sorry, I got confused by the title of the PR, looks like you mention other bugs in the commit message :)